lemur/plugins/lemur_azure_dest/plugin.py

```diff
@@ -0,0 +1,191 @@
+"""
+.. module: lemur.plugins.lemur_azure_dest.plugin
+    :platform: Unix
+    :copyright: (c) 2019
+    :license: Apache, see LICENCE for more details.
+
+    Plugin for uploading certificates and private key as secret to azure key-vault
+     that can be pulled down by end point nodes.
+
+.. moduleauthor:: sirferl
+"""
+from flask import current_app
+
+from lemur.common.defaults import common_name, bitstrength
+from lemur.common.utils import parse_certificate, parse_private_key
+from lemur.plugins.bases import DestinationPlugin
+
+from cryptography.hazmat.primitives import serialization
+import requests
+import json
+import sys
+import base64
+
+
+def base64encode(string):
```

Comment:
I wonder if this should move into util

https://github.com/Netflix/lemur/blob/bde3d1ac66a50c91cf8bc6daab001b475115e1d9/lemur/plugins/lemur_kubernetes/plugin.py#L78-L79

Reply:
moved this function to common.utils